charts/kube-prometheus-stack/Chart.yaml

```diff
@@ -18,7 +18,7 @@ name: kube-prometheus-stack
 sources:
   - https://github.com/prometheus-community/helm-charts
   - https://github.com/prometheus-operator/kube-prometheus
-version: 18.1.0
+version: 18.2.0
```

Comment:
As this changes the current value i would consider this a breaking change. Therefor major version change needed.

```suggestion
version: 19.0.0
```

Reply:
Makes sense, changed it.